src/documents/tables/types/indexes/create-index.ts

```diff
@@ -17,7 +17,7 @@ import type { CommandOptions } from '@/src/lib/index.js';
 /**
  * @public
  */
-export type TableCreateIndexColumn<WSchema> = keyof WSchema | Partial<Record<(keyof WSchema & string), `$${string}`>>;
+export type TableIndexColumn<WSchema> = keyof WSchema | Partial<Record<(keyof WSchema & string), '$keys' | '$values'>>;

```

Comment:
This rename is a backwards breaking change and ideally should have been made in a  major release